docker compose for dev

jonatron opened the pull request:
docker-compose makes it easier to work on

yoe commented:
Hardcoding a config file like this means that whenever we update the configuration to add a new option, the dev config file needs to be updated to match. This is a PITA.

I'm not familiar enough with docker-compose to know whether this is possible, but I would prefer if we could set up the docker-compose file so that it runs `sreview-config -a update` some time at startup. This will generate a config file in the default location if one does not yet exist.

If that can't be done, then I'm happy to merge as is, except that I'd like to then add a message at the top that explains that the config file is necessarily outdated and that running `sreview-config -a update` would help.

I would also appreciate if you could add a small section to the README documentation explaining how to start using docker-compose. This doesn't have to be too detailed, and can point to docs for the nitty gritty details, just should explain how to get started.

jonatron commented:
I've updated it to use sreview-config to generate the config file rather than including a copy. Added a small bit to "Development" in README. Hopefully you or someone else can check it works for them and not just me.
